lib/framework/widget/view_util.dart

```diff
@@ -73,6 +74,10 @@ class ViewUtil {
         payload = item[widgetType];
       }
       def = getDefinition(item);
+      if ( item.keys.length > 1 ) {
```

Comment:
ignore this, it is a no-op and will be in the next change, idea is to detect common issues with tabbing in yaml